code/korniakov-kirill-lab1/src/test/java/ru/unn/agile/ComplexNumber/Model/SizovaKsenyaTest.java

```diff
@@ -0,0 +1,78 @@
+package ru.unn.agile.ComplexNumber.Model;
+
+import org.junit.Test;
+import ru.unn.agile.ComplexNumber.model.ComplexNumber;
+
+import static org.junit.Assert.*;
+
+public class SizovaKsenyaTest {
+    private final double delta = 0.001;
+
+    @Test
+    public void canCreateDefaultComplexNumber() {
+        ComplexNumber complexNumber = new ComplexNumber();
+
+        // Assert
+        assertEquals(complexNumber.getReal(), 0, delta);
+        assertEquals(complexNumber.getImaginary(), 0, delta);
+    }
+
+    @Test
+    public void canCreateComplexNumberWithValues() {
+        ComplexNumber complexNumber = new ComplexNumber(1, 2);
+
+        // Assert
+        assertEquals(complexNumber.getReal(), 1, delta);
+        assertEquals(complexNumber.getImaginary(), 2, delta);
```

Comment:
You use a lot of magic numbers, like here, fix it please.
it should look something like this:

        // Arrange
        ComplexNumber complexNumber = new ComplexNumber(1, 2);
        var expectedRealPart = 1;
        var expectedImagPart = 2;

        // Assert
        assertEquals(complexNumber.getReal(), expectedRealPart, delta);
        assertEquals(complexNumber.getImaginary(), expectedImagPart, delta);